### PR TITLE
Add step on building NuGet package

### PR DESCRIPTION
**Pull request by Kent1LG**
## Overview
About this last issue : #6993 

With the aim to make the documentation more intuitive i propose a small addition :
Add a last step to explicitly ask to the user to press the save button when they add a new NugGet source on properties.

## Changes
- Add a "Save" step
- Add an indicative image

### CODE REVIEW

**Comment by wiwei** [comment]
Totally makes sense, and also definitely something that would be super easy for folks to overlook (and then hit issues as you've hit).

Thanks for making the docs better!

**Comment by wiwei** [comment]
Approving conditionally on the image change - this is great, thanks again

**Comment by Kent1LG** [comment]
Hi Will, there is a proposition of image, what do you think? 
I m actually using unity personnal so i don't have the Dark theme, sorry (this is the reason i didn't add the new source in the previous image ahah).

File: Documentation/MRTKNuGetPackage.md [hunk 1]
```diff
@@ -73,6 +73,9 @@ With the latest source from MRTK, you can build the NuGet package locally and co
     - Press **Add New Source** and replace **source_path** with the `<Path to your Repository>\NuGet\artifacts`
 
         ![Edit Preferences Menu Item](Images/NuGet/AddNewSource.png)
+    - At the bottom, press **Save** button.
```
Comment (thalbern) [inline Documentation/MRTKNuGetPackage.md]:
```suggestion
    - At the bottom, press the **Save** button.
```

**Comment by wiwei** [comment]
@thalbern got to it before me, but thanks for updating it @Kent1LG. Looks great.

**Pull request merged**